src/variable_parser.cpp

```diff
@@ -86,18 +96,24 @@ void parse_variables(char* filename){
 			cout << "No file: " << filename << "\n";
 		throw(1);
 	}
+
 	ifstream in(filename);
 	string line;
 	while(getline(in, line)){
 		stringstream ss1(line);
 		if(getline(ss1, line, ';')){
 			string variable, value;
-			line.erase(remove(line.begin(), line.end(), ' '), line.end());
+				
+			if (strcmp(filename,"storage_location") != 0) {

```

Comment:
`if(!strcmp(...)){` , though why don't we want to strip lines in storage_location?

Reply:
If a user's storage_location directory contains spaces, then these could be stripped by variable_parser and result in the incorrect path being defined.

Reply:
Ah fair, hmm, really need a better variable parser. I'm hoping to convert it to json at some point